server/index.js

```diff
@@ -0,0 +1,12 @@
+const { port } = require('./config');
+const open = require('./open');
+const express = require('express');
+const app = express();
+
+app.use(express.static('src'));
+app.use(express.static('server'));
+
+app.listen(port, () => {
+    console.log(`Example app listening on port ${port}!`)
+    open(`http://localhost:${port}`)
+});
```

Comment:
This file is the code for the server.  The line `const express = require('express')` pulls in a massive amount of code other engineers have created inio this file.  

The rest is code to configure the server, telling it to serve up the files we make.